### PR TITLE
Add manual enrollment order API method

### PR DESCRIPTION
**Pull request by tuchfarber**
**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
- [ ] Version pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)

### CODE REVIEW

File: enterprise/api_client/ecommerce.py [hunk 1]
```diff
@@ -65,3 +65,41 @@ def get_course_final_price(self, mode, currency='$', enterprise_catalog_uuid=Non
...
+                str(exc)
+            )
+        order_creations = order_repsonse["orders"]
+        successful_creations = [order for order in order_creations if order["status"] == "success"]
+        failed_creations = [order for order in order_creations if order["status"] == "failure"]
+        LOGGER.info(
+            "Successfully created orders for the following manual enrollments. %s",
+            successful_creations
+        )
+        LOGGER.error(
```
Comment (tuchfarber) [inline enterprise/api_client/ecommerce.py]: Is logging here enough to raise awareness if orders haven't been created? I want to make sure we notice if there's failures.

File: enterprise/__init__.py [hunk 2]
```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "2.0.9"
+__version__ = "2.0.10"
```
Comment (crice100) [inline enterprise/__init__.py]: Minor: also update the comment describing this version

File: enterprise/api_client/ecommerce.py [hunk 3]
```diff
@@ -65,3 +65,41 @@ def get_course_final_price(self, mode, currency='$', enterprise_catalog_uuid=Non
...
+        except (SlumberBaseException, ConnectionError, Timeout) as exc:
+            LOGGER.exception(
+                "Failed to create order for manual enrollments for the following enrollments: %s. Reason: %s",
+                enrollments,
+                str(exc)
+            )
+        order_creations = order_repsonse["orders"]
+        successful_creations = [order for order in order_creations if order["status"] == "success"]
+        failed_creations = [order for order in order_creations if order["status"] == "failure"]
+        LOGGER.info(
```
Comment (crice100) [inline enterprise/api_client/ecommerce.py]: Minor: maybe only log these messages if there are successful/failed creations

File: enterprise/api_client/ecommerce.py (hunk 1 above)
Comment (crice100) [reply enterprise/api_client/ecommerce.py]: I'm not sure. We could probably check opsgenie and make a new alert if we can't find any existing generic error checks

**Comment by tuchfarber** [comment]
I was having testing issues and made a separate branch, then I ended up fixing the tincan and CHANGELOG stuff in the other branch 🤦‍. Carried over the changes from that branch and made the logging optional.

**Comment by tuchfarber** [comment]
Re: testing, I think it's just a passthrough layer to another service so it's not worth testing since we would mostly just be testing the enterprise API client. We can add testing to the places we call it to make sure they're shuttling the data correctly.

File: enterprise/api_client/ecommerce.py [hunk 4]
```diff
@@ -65,3 +65,43 @@ def get_course_final_price(self, mode, currency='$', enterprise_catalog_uuid=Non
...
+                    "enrollments": enrollments
+                }
+            )
+        except (SlumberBaseException, ConnectionError, Timeout) as exc:
+            LOGGER.exception(
+                "Failed to create order for manual enrollments for the following enrollments: %s. Reason: %s",
+                enrollments,
+                str(exc)
+            )
+        order_creations = order_repsonse["orders"]
```
Comment (HammadAhmadWaqas) [inline enterprise/api_client/ecommerce.py]: @tuchfarber `order_repsonse` won't be declared if there will be an exception  in `manual_course_enrollment_order`
I think you want to return after logging exception?

**Pull request merged**